scripts/check_version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+ROOT_PATH=`dirname "${0}"`/..
+
+IOS_VERSION=$(awk <"$ROOT_PATH/ios/manifest" "\$1 == \"version:\" { print \$2 }")
+ANDROID_VERSION=$(awk <"$ROOT_PATH/android/manifest" "\$1 == \"version:\" { print \$2 }")
+
+if [ $IOS_VERSION != $ANDROID_VERSION ]; then
```

Comment:
Do you need quotes around these vars in case they are empty or have a space in them?